pkg/utils/test/test.go

```diff
@@ -121,3 +122,21 @@ func WithWd(path string) func() {
 		}
 	}
 }
+
+// WithFeatureGate sets the specified gate to the specified value, and returns a function that restores the original value.
+// Failures to set or restore cause the test to fail.
+// Example use:
+// defer WithFeatureGate(utilfeature.DefaultFeatureGate, features.<FeatureName>, true)()
```

Comment:
Nit, example code in doc strings should be indented:
```suggestion
//   defer WithFeatureGate(utilfeature.DefaultFeatureGate, features.<FeatureName>, true)()
```

Reply:
Done